azplugins/bond.py

```diff
@@ -10,6 +10,67 @@
 
 from . import _azplugins
 
+class double_well(hoomd.md.bond._bond):
+    R""" double well bond potential.
+
+    Args:
+        name (str): Name of the bond instance.
+
+    :py:class:`double_well` specifies a double well potential between the two particles in each defined bond.
+    The potential is given by:
+
+    .. math::
+        :nowrap:
+
+        \begin{eqnarray*}
+        V_{\mathrm{DW}}(r)  =  \frac{V_{max}}{b^4} \left[ \left( r - a/2 \right)^2 - b^2 \right]^2
+        \end{eqnarray*}
+
+    Coefficients:
+
+    - :math:`V_max` - Potential maximum height between the two minima at 1/2``a`` (in energy units)
```

Comment:
Should this be:
```
``a``/2
```
?

Similarly below. If so, prefer putting the constant denominator after the numerator so that it is not ambiguous.